src/ResourceManager/Compute/Commands.Compute/AvailabilitySets/GetAzureAvailabilitySetCommand.cs

```diff
@@ -60,6 +86,19 @@ public override void ExecuteCmdlet()
                         psResultList.Add(psItem);
                     }
 
+                    var nextPageLink = result.Body.NextPageLink;
```

Comment:
if there's an existing test case for this, do we need to re-record it because of the use of paginatin now?